### PR TITLE
Check the path to CC and CXX when building libUnwind

### PR DESCRIPTION
**Pull request by zachriggle**
Closes google/honggfuzz#159

### CODE REVIEW

**Comment by zachriggle** [comment]
CLA signed, but also a Google employee

File: third_party/android/scripts/compile-libunwind.sh [hunk 1]
```diff
@@ -163,10 +163,21 @@ HOST_OS=$(uname -s | tr '[:upper:]' '[:lower:]')
...
-export CC="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-gcc --sysroot=$SYSROOT"
-export CXX="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-g++ --sysroot=$SYSROOT"
+export CC="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-gcc"
+export CXX="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-g++"
 export PATH="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin":$PATH
 
+if [ ! -f "$CC" ]; then
+  echo "[-] gcc doesn't exist: $CC"
+  abort 1
+elif [ ! -f "$CXX" ]; then
```
Comment (robertswiecki) [inline third_party/android/scripts/compile-libunwind.sh]: Maybe -x

File: third_party/android/scripts/compile-libunwind.sh [hunk 2]
```diff
@@ -163,10 +163,21 @@ HOST_OS=$(uname -s | tr '[:upper:]' '[:lower:]')
 HOST_ARCH=$(uname -m)
 
 SYSROOT="$NDK/platforms/$ANDROID_API/arch-$ARCH"
-export CC="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-gcc --sysroot=$SYSROOT"
-export CXX="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-g++ --sysroot=$SYSROOT"
+export CC="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-gcc"
+export CXX="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin/$TOOLCHAIN-g++"
 export PATH="$NDK/toolchains/$TOOLCHAIN_S/prebuilt/$HOST_OS-$HOST_ARCH/bin":$PATH
 
+if [ ! -f "$CC" ]; then
```
Comment (robertswiecki) [inline third_party/android/scripts/compile-libunwind.sh]: As for CXX (-x)

File: third_party/android/scripts/compile-libunwind.sh (hunk 1 above)
Comment (zachriggle) [reply third_party/android/scripts/compile-libunwind.sh]: Done

File: third_party/android/scripts/compile-libunwind.sh (hunk 2 above)
Comment (zachriggle) [reply third_party/android/scripts/compile-libunwind.sh]: Done

**Pull request merged**